packages/headless/src/app/headless-engine.ts

```diff
@@ -222,6 +225,43 @@ export class HeadlessEngine<Reducers extends ReducersMapObject>
     }
   }
 
+  private validateConfiguration(options: HeadlessOptions<Reducers>) {
+    const configurationSchema = new Schema<HeadlessConfigurationOptions>({
```

Comment:
I only validated the configuration since it was the only thing that made sense to validate.